client/app/components/empty-state/empty-state.js

```diff
@@ -45,6 +46,12 @@ const EmptyStateComponent = {
         },
       });
     };
+
+    $scope.verifyEmail = () => {
```

Comment:
You don't need `$scope` here. Just put a method on `this` and in the template, use `$ctrl.verifyEmail`.

Reply:
![image](https://user-images.githubusercontent.com/289488/51083949-d9b39100-172a-11e9-8f2b-22fba9374787.png)